packages/editor/src/components/plugin-more-menu-item/index.js

```diff
@@ -74,7 +67,7 @@ export default compose(
 		return {
 			as: ownProps.as ?? MenuItem,
 			icon: ownProps.icon || context.icon,
-			name: 'core/edit-site/plugin-more-menu',
+			name: 'core/editor/plugin-more-menu',
```

Comment:
It seems there will be a behavior change, a plugin that included code with a more menu item only for the site editor, will now also load that more menu on the post editor. Maybe it is an acceptable change but it is something we should document on the dev not that is planned.

Reply:
Yes, this will be the same for all the extensibility APIs. It has been discussed in https://github.com/WordPress/gutenberg/pull/59985 we think it's fine but we'll be monitoring feedback.